spec/features/region_admin/donors_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+RSpec.describe "Region Admin Donors" do
+
+  feature "Viewing the list of donors" do
+    let(:boulder) { create(:region) }
+    let(:denver)  { create(:region) }
+
+    let!(:boulder_donor) { create(:donor, region: boulder, name: "Boulder donor site") }
+    let!(:denver_donor)  { create(:donor, region: denver,  name: "Denver donor site") }
```

Comment:
I see your point.  Pretty readable either way and I don't have a strong preference.  Leaving as is.